checks/container.go

```diff
@@ -57,7 +57,6 @@ func (c *ContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.Me
 
 	// Fetch orchestrator metadata once per check.
 	ecsMeta := ecs.GetMetadata()
```

Comment:
can ecs be dropped too?

Reply:
It doesn't look like the ecs tagger collector supports Fargate at the moment, so I was hesitant to remove it as it'd be a slight regression. I'm not 100% sure if it'll be added as part of DataDog/datadog-agent#847.

Reply:
OK that's fair, thanks. We can leave in until that's ready
cc @hkaj